[Pipeline] CI Build Failure on main: error CS1002: ; expected [/home/runner/work/prd-to-prod/prd-to-prod/TicketDeflec

## CI Build Failure on `main`

- **Run**: https://github.com/samuelkahessay/prd-to-prod/actions/runs/22582509313
- **Branch**: `main`
- **Commit**: `fc962123651e44e599cff68474b5f9a43ced952c`
- **Failure Type**: build
- **Failure Signature**: `cs1002-expected`

### Failure Summary
error CS1002: ; expected [/home/runner/work/prd-to-prod/prd-to-prod/TicketDeflection/TicketDeflection.csproj]

### Failed-Step Excerpt
```text
deploy-azure / build-and-deploy	UNKNOWN STEP	2026-03-02T15:20:41.2035116Z ##[error]/home/runner/work/prd-to-prod/prd-to-prod/TicketDeflection/Canary/DrillCanary.cs(11,46): error CS1002: ; expected [/home/runner/work/prd-to-prod/prd-to-prod/TicketDeflection/TicketDeflection.csproj]
deploy-azure / build-and-deploy	UNKNOWN STEP	2026-03-02T15:20:41.2746521Z Build FAILED.
deploy-azure / build-and-deploy	UNKNOWN STEP	2026-03-02T15:20:41.2750202Z ##[error]/home/runner/work/prd-to-prod/prd-to-prod/TicketDeflection/Canary/DrillCanary.cs(11,46): error CS1002: ; expected [/home/runner/work/prd-to-prod/prd-to-prod/TicketDeflection/TicketDeflection.csproj]
```

> This issue was created automatically by the CI Failure Router workflow.

<!-- self-healing-drill:v1
drill_id=20260302-152002
observed_commit=fc962123651e44e599cff68474b5f9a43ced952c
observed_failure_run_id=22582509313
observed_failure_run_url=https://github.com/samuelkahessay/prd-to-prod/actions/runs/22582509313
-->

--- a/TicketDeflection/Canary/DrillCanary.cs
+++ b/TicketDeflection/Canary/DrillCanary.cs
@@ -8,5 +8,5 @@ namespace TicketDeflection.Canary;
 /// </summary>
 public static class DrillCanary
 {
-    public static string Status() => "broken"
+    public static string Status() => "broken";
 }

